packages/web-component-library-stencil/src/components/button.tsx

```diff
@@ -122,6 +122,9 @@ export class Button {
       button.value = value || '';
     }
 
+    const hasIconSlot = this.hostElement.querySelector('[slot="icon"]');
+    const hasLabelSlot = this.hostElement.querySelector('[slot="label"]');
```

Comment:
Zou je willen testen of deze oplossing ook werkt als je achteraf een `<utrecht-icon slot="icon">` toevoegt?

Reply:
@Robbert 

Wat bedoel je precies met achteraf? De story werkt momenteel met dit gebruik:
```
 <UtrechtButton {...props}>
      <span slot="icon">
        <IconElement />
      </span>
    )}
    ... label & children
</UtrechtButton>
 ```

Bedoel je iets in deze richting?
```
 <UtrechtButton {...props}>
      <utrecht-icon-example slot="icon" />
    )}
    ... label & children
</UtrechtButton>
 ```

Reply:
<img width="871" alt="image" src="https://github.com/user-attachments/assets/d8c1126c-a54b-494c-9204-ef9fab02172b">